tests/unit/plugins/modules/test_kutils.yaml

```
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) Alexei Znamensky (russoz@gmail.com)
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+---
+- id: test_kinit_default
+  input:
+    state: present
+    password: cool_password
+  output:
+    changed: true
+  run_command_calls:
+    - command: [/testbin/klist]
+      environ: &env-def {environ_update: {LANGUAGE: C, LC_ALL: C}, check_rc: true}
+      rc: 1
+      out: ""
+      err: ""
+    - command: [/testbin/kinit]
+      environ: *env-def
+      rc: 0
+      out: ""
+      err: ""
```

hi @abakanovskii ,

My bad - I told you about the test helper and forgot to mention to you about an upcoming change. My PR https://github.com/ansible-collections/community.general/pull/8894 was merged 4 days ago and changed slightly the formatting, so this block here should look like:
```suggestion
  mocks:
    run_command:
    - command: [/testbin/klist]
      environ: &env-def {environ_update: {LANGUAGE: C, LC_ALL: C}, check_rc: true}
      rc: 1
      out: ""
      err: ""
    - command: [/testbin/kinit]
      environ: *env-def
      rc: 0
      out: ""
      err: ""
```
repeated throughout the file

---

then it's clear why i missed that haha, thanks for the info!